src/components/LineChart.jsx

```diff
@@ -13,21 +13,13 @@ const calculateHeight = (total) => {
 };
 
 const calculateMinMax = (data) => {
-    const lastValue = data[data.length-1];
-    let maxValue = undefined;
-    let minValue = undefined;
-    if (lastValue < 100) maxValue = 250;
-    else if (lastValue < 250) maxValue = 500;
-    else if (lastValue < 500) maxValue = 1000;
-    else if (lastValue < 1000) maxValue = 2000;
-    else if (lastValue < 2500) maxValue = 5000;
-    else if (lastValue < 7500) maxValue = 10000;
-
-    if (lastValue > 5000) minValue = 0;
```

Comment:
👎 Removing this is a bad idea, as the graphs are now very misleading. I've been so confused about the insane jumps in popularity in some plugins — how did https://plugins.jenkins.io/git-parameter/ **double** its installations!? Wait… it didn't. It's like +10% and graphs are now terrible.

If there is no easily visible difference in numbers if you don't show Y=0, then there are no changes that matter. The way we collect this data is unreliable in the first place, so if you want to discern differences between a few percentages, you're already doing it wrong.

Reply:
Amended in #445.